src/data/navigation.ts

```diff
@@ -343,6 +344,7 @@ const navigation_5_2: NavigationItem[] = [
               { title: "Cody for Neovim", href: "/cody/clients/install-neovim", },
               { title: "Cody for Web", href: "/cody/clients/cody-with-sourcegraph", },
               { title: "Cody for Enterprise", href: "/cody/clients/enable-cody-enterprise", },
+              { title: "LLM Model Configuration", href: "/cody/clients/model-configuration", },
```

Comment:
@MaedahBatool As a meta point, the docs and layout in the "Enabling Cody Enterprise" page is getting very long. Let's start thinking about better organizing the data after Aug 7th :)

Reply:
I agree with you 100% I have plans to revamp the entire Cody docs by Q3 with a better layout.

Reply:
I added a project for Merge to get this rolling :)